gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Dec 28 10:00:20 PST 2015
+#Fri May 12 17:06:15 EDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```

Comment:
Gradle should not affect the version of Android unless you change build.gradle under the app folder. It is simply responsible for dependencies, building, and tasks (perhaps a simplification of it's capabilities).  It might affect customized tasks in your build.gradle file, but it does not look like you have many of those yet.
That being said you certainly want _everyone on the same page_ for what gradle version the project will use.  
Sorry I have not introduced myself. I'm David Smith. I'm with SIL-PNG in Ukarumpa.